### PR TITLE
Stop iniswapping from modifying the char.ini file

### PR DESCRIPTION
**Pull request by Crystalwarrior**
Stop iniswapping from modifying the char.ini (which can potentially remove comments such as credits, charmaker notes, etc.)
It also lead to really annoying behavior where if you swapped out from that character, when you swap back you'd be put into the iniswap instead of the default state, which could even be outside that character's iniswap list.

Closes https://github.com/AttorneyOnline/AO2-Client/issues/737

### CODE REVIEW

**Review by oldmud0 — COMMENTED** [review]
I'm glad we are removing one of the most nerve wracking parts of courtroom.cpp. The function that returns a name given a name

**Review by stonedDiscord — APPROVED** [review]
it did WHAT before

**Review by stonedDiscord — DISMISSED** [review]

**Comment by stonedDiscord** [comment]
what happened with `get_char_name`

File: src/text_file_functions.cpp [hunk 1]
```diff
@@ -589,15 +590,6 @@ QStringList AOApplication::read_ini_tags(VPath p_path, QString target_tag)
...
 
-QString AOApplication::get_char_name(QString p_char)
-{
-  QString f_result = read_char_ini(p_char, "name", "Options");
-
-  if (f_result == "")
-    return p_char;
-  return f_result;
-}
-
```
Comment (oldmud0) [inline src/text_file_functions.cpp:600]: I'd like to accept this PR but `get_char_name` is still depended on according to the linker. Please bring it back or remove references to it.

**Review by oldmud0 — CHANGES_REQUESTED** [review]

File: src/courtroom.cpp [hunk 2]
```diff
@@ -4441,11 +4447,9 @@ void Courtroom::set_iniswap_dropdown()
   }
   QStringList iniswaps =
       ao_app->get_list_file(ao_app->get_character_path(char_list.at(m_cid).name, "iniswaps.ini")) +
+      ao_app->get_list_file(VPath("iniswaps.ini")) +
       ao_app->get_list_file(ao_app->get_base_path() + "iniswaps.ini");
```
Comment (TrickyLeifa) [inline src/courtroom.cpp:4451]: This causes duplicate entries when editing the iniswaps.ini
[bugs.webm](https://user-images.githubusercontent.com/26681464/181112303-8a31c5e6-f169-477e-ba78-84264e535d25.webm)
.

**Review by TrickyLeifa — DISMISSED** [review]

**Review by oldmud0 — APPROVED** [review]
Good riddance

**Pull request merged**